### DISCUSSION TITLE
Validation not working with React-Select (No Yup)

### DISCUSSION THREAD

**Discussion by gustavomm19** [react-hook-form/react-hook-form#3970] [Q&A] [answered]
Hi! (its me again) I am using a library called react-timezone-select (which is totally absed on react-select) and validation with Controller is not working :(

I saw someone with a similar issue here, but he was using Yup and I am not. How could I fix this?

I need help

Sandbox here:
https://codesandbox.io/s/help-with-react-select-and-form-hook-ooh9x

**Comment by bluebill1049** [comment, marked as answer]
Take a look at the doc: 

```
<Controller 
              as={TimezoneSelect} 
              name="timeZone" 
              control={control} 
              onChange={([selected]) => {
                // React Select return object instead of value for selection
                return { value: selected };
              }}
              defaultValue={{}}
              ref={register({ required: true })}
            />
```
            
for controller you should use `rules` and its taking care the `register` for you.

**Reply by gustavomm19** [reply]
Thanks!!
